static/docs/command-reference/get.md

```diff
@@ -32,31 +36,28 @@ external <abbr>project</abbr>. Both HTTP and SSH protocols are supported for
 online repositories (e.g. `[user@]server:project.git`). `url` can also be a
 local file system path to an "offline" repository.
 
-The `path` argument of this command is used to specify the location of the data
-to be downloaded within the source project. It should point to a data file or
-directory tracked by that project – specified in one of the
-[DVC-files](/doc/user-guide/dvc-file-format) of the repository at `url`. (You
-will not find these files directly in the source Git repository.) The source
-project should have a default [DVC remote](/doc/command-reference/remote)
-configured, containing them.)
+The `path` argument of this command is used to specify the location of the file
+or directory within the source project. If the file is a
+[DVC-file](/doc/user-guide/dvc-file-format) the source project must have a
+default [DVC remote](/doc/command-reference/remote) configured.
 
-> See `dvc get-url` to download data from other supported URLs.
+> See `dvc get-url` to obtain data from other supported URLs.
```

Comment:
In this case `dvc get-url` has nothing to do with Git so "download" is correct and precise here, I think.